drivers/regulator/regulator_cp9314.c

```diff
@@ -230,6 +236,32 @@ static struct cp9314_reg_patch otp_1_patch[3] = {
 	{CP9314_REG_TSBAT_CTRL, CP9314_LB1_STOP_PHASE_SEL, CP9314_LB1_STOP_PHASE_SEL},
 };
 
+static int regulator_cp9314_check_faults(const struct device *dev)
```

Comment:
Is the user able to access this information?
Could you link this to the `get_error_flags` API instead?